controllers/user.js

```diff
@@ -110,4 +110,29 @@ module.exports = {
       .then((user) => promisifyObjCallback(req, 'logIn')(user))
       .then(() => ({ status: 'OK', message: 'Your password has been changed.' }));
   }),
+  getProfile(req, res) {
+    res.render('account/profile');
+  },
+  postUpdateProfile: promisifyJSON((req, res) => {
+    const errors = req.validationErrors();
+    if (errors) {
+      req.flash('errors', errors);
+      return res.redirect('/account/profile');
```

Comment:
No, I don't use ajax here. I submit a form normally. Do you think we need to use ajax?